### PR TITLE
Implement contributors view

### PR DESCRIPTION
**Pull request by srym**
## Issue
- #78

## Overview (Required)
- Implemented anyway
- Any comment is welcomed as I'm not so used to MVVM & Data Binding, Thanx!

## Links
-

## Screenshot
Before | After
:--: | :--:
 | contributors

### CODE REVIEW

File: app/src/main/res/layout/view_contributor_cell.xml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+            android:clickable="true"
+            android:onClick="@{viewModel::onClickContributor}"
+            >
+
+        <ImageView
+                android:id="@+id/avatar"
+                android:layout_width="@dimen/contributor_image"
+                android:layout_height="@dimen/contributor_image"
+                android:layout_centerHorizontal="true"
+                android:layout_margin="@dimen/contributor_cell_margin"
```
Comment (konifar) [inline app/src/main/res/layout/view_contributor_cell.xml]: Can you change margin bottom to `8dp`? I want to show more contributors in one page.

File: app/src/main/res/layout/view_contributor_cell.xml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+                app:speakerImageSize="@{@dimen/contributor_image}"
+                app:speakerImageUrl="@{viewModel.avatarUrl}"
+                />
+
+        <TextView
+                android:id="@+id/name"
+                style="@style/TextBody1"
+                android:layout_width="match_parent"
```
Comment (konifar) [inline app/src/main/res/layout/view_contributor_cell.xml]: Can you add `textColor="@color/black"`?

File: app/src/main/res/layout/view_contributor_cell.xml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+                android:layout_height="wrap_content"
+                android:layout_below="@id/avatar"
+                android:layout_marginBottom="@dimen/contributor_cell_margin"
+                android:gravity="center"
+                android:lines="2"
```
Comment (konifar) [inline app/src/main/res/layout/view_contributor_cell.xml]: Sorry, I told you this `lines="2"`, but can you change this to `maxLines="2"`? 🙇

Comment (konifar) [inline app/src/main/res/layout/view_contributor_cell.xml]: Plus, please add `android:ellipsize="end"`.

**Comment by konifar** [comment]
@srym Reviewed! Almost perfect! Thanks for contribution!

File: app/src/main/res/layout/view_contributor_cell.xml (hunk 1 above)
Comment (srym) [inline app/src/main/res/layout/view_contributor_cell.xml]: @konifar Do you mean to change only ImageView's margin bottom to 8dp while keeping margin top, left, right 16dp?
I also have a TextView under the ImageView with its margin bottom 16dp. What do you think of this margin bottom?

Just FYI, below is *all* margins are set to 8dp including ImageView's margins and TextView's margin bottom.
contributors_design_fix

Comment (konifar) [inline app/src/main/res/layout/view_contributor_cell.xml]: > Do you mean to change only ImageView's margin bottom to 8dp while keeping margin top, left, right 16dp?

Yes, I want to change the margin between ImageView and TextView!

implement_contributors_view_by_srym_ _pull_request__138_ _droidkaigi_conference-app-2017

Comment (srym) [inline app/src/main/res/layout/view_contributor_cell.xml]: @konifar The image below is ImageView's margin top, left, right is 16dp and bottom is 8dp.
TextView's margin bottom is also 16dp. Your wish is my command, sir ;)

contributors_design_fix02

**Comment by srym** [comment]
@konifar 
Fixed & pushed! I'm glad if you take a look at later!
9458bc27bf5cad22476127193730a2ca8e1d6f25

**Comment by konifar** [comment]
Perfect 💯

**Pull request merged**